build.gradle

```diff
@@ -39,31 +50,78 @@ repositories {
+    flinkShadowJar.exclude group: 'com.google.code.findbugs', module: 'jsr305'
+    flinkShadowJar.exclude group: 'org.slf4j'
+    flinkShadowJar.exclude group: 'log4j'
+}
+
+configurations.all {
+    resolutionStrategy {
+        force "com.google.guava:guava:" + guavaVersion
+    }
+}
+
+shadowJar {
+    configurations = [project.configurations.flinkShadowJar]
+}
+
+lombok {
+    version = lombokVersion
+}
 
 dependencies {
     // --------------------------------------------------------------
     // Compile-time dependencies that should NOT be part of the
     // shadow jar and are provided in the lib folder of Flink
     // --------------------------------------------------------------
-    compileOnly "org.apache.flink:flink-java:${flinkVersion}"
-    compileOnly "org.apache.flink:flink-streaming-java_${flinkScalaVersion}:${flinkVersion}"
-    compileOnly "org.apache.flink:flink-table-api-java-bridge_${flinkScalaVersion}:${flinkVersion}"
-    compileOnly "org.apache.flink:flink-table-planner_${flinkScalaVersion}:${flinkVersion}"
-    compileOnly "io.pravega:pravega-connectors-flink-${flinkShortVersion}_${flinkScalaVersion}:${pravegaFlinkConnectorVersion}"
-    compileOnly "io.pravega:pravega-client:0.11.0"
-    compileOnly "io.pravega:pravega-keycloak-client:${pravegaCredentialsVersion}"
-
-    compileOnly "org.slf4j:slf4j-api:${slf4jApiVersion}"
-    compileOnly "ch.qos.logback:logback-classic:${logbackVersion}"
-    compileOnly "ch.qos.logback:logback-core:${logbackVersion}"
-    compileOnly "org.scala-lang:scala-compiler:${scalaCompilerVersion}"
-
-    testCompileOnly "org.apache.flink:flink-test-utils_${flinkScalaVersion}:${flinkVersion}"
+    compileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: "${spotbugsAnnotationsVersion}"
+    testCompileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: "${spotbugsAnnotationsVersion}"
+    compileOnly 'org.projectlombok:lombok:' + "${lombokVersion}"
+    testCompileOnly 'org.projectlombok:lombok:' + "${lombokVersion}"
+    annotationProcessor 'org.projectlombok:lombok:' + "${lombokVersion}"
+    testAnnotationProcessor 'org.projectlombok:lombok:' + "${lombokVersion}"
+    implementation "org.apache.flink:flink-java:${flinkVersion}"
+    implementation "org.apache.flink:flink-streaming-java:${flinkVersion}"
+    implementation "org.apache.flink:flink-connector-kafka:${flinkVersion}"
+    implementation "org.apache.flink:flink-table-api-java-bridge:${flinkVersion}"
+    implementation "org.apache.flink:flink-table-planner_${flinkScalaVersion}:${flinkVersion}"
+    implementation "org.apache.flink:flink-clients:${flinkVersion}"
+    implementation "org.apache.flink:flink-runtime-web:${flinkVersion}"
+    implementation "org.apache.flink:flink-core:${flinkVersion}"
+//    implementation "io.pravega:pravega-connectors-flink-${flinkShortVersion}_${flinkScalaVersion}:${pravegaFlinkConnectorVersion}"
```

Comment:
Please, clean up from this file all the unnecessary imports and comments like this.